src/styles/blocks/button.scss

```diff
@@ -0,0 +1,31 @@
+.button {
+  width: 278px;
+  height: 56px;
+  border: none;
+  border-radius: 40px;
+  background-color: #fff;
+  color: #1d1d1d;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 27px;
+  letter-spacing: 0;
+  text-align: center;
+  transition: box-shadow $animation-standard-duration ease;
```

Comment:
consider adding empty lines between big blocks of css properties

Reply:
Hello @jv-aquino,

About checklist 10- I checked my code and I have my html tag with property scroll-behavior: smooth;(see src/styles/main.scss). I checked DemoLink and everything is ok, let me know if in your browser or tests this is not working correctly.

Regarding image sizes, I think your browser may have a zoom setting that changes the responsiveness of the page, I checked it in my browser with 100% zoom, let me know if you are evaluating any specific situation of media beyond width of screen. 
![image](https://github.com/mate-academy/layout_landing-page/assets/151199792/f5a161e2-3967-4e85-9e8c-7c1f0c2857d3)

I submitted the PR again without changes (I see and agree with improvements in hover transitions), however we have a deadline today for this activity, and if you guarantee that the landing page has no critical issues, I would rather finish this task.
We have some issues with matescripts (it looks like some updates have already been done after this PR) and I might miss deadlines facing issues with tests and scripts.